spec/core_functions/meta/get_mixin/content.hrx

```diff
@@ -0,0 +1,172 @@
+<===> passes-content/input.scss
+@use "sass:meta";
+
+@mixin foo {
+    @content;
+}
```

Comment:
It's probably a good idea to have a wrapper style rule here to be clear that the `@content` isn't being passed through as-is.